drivers/i2s/i2s_nrfx.c

```diff
@@ -0,0 +1,1309 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @brief Driver for Nordic Semiconductor nRF I2S
+ */
+
+
+#include <string.h>
+#include <stdlib.h>
+#include <dma.h>
+#include <i2s.h>
+#include <soc.h>
+#include <nrfx.h>
+#include <nrfx_i2s.h>
+#include "i2s_nrfx.h"
+#define LOG_DOMAIN "i2s_nrfx"
+#include <logging/log.h>
+#define LOG_LEVEL _LOG_LEVEL_DBG
+
+LOG_MODULE_REGISTER(i2s_nrfx);
+
+#define LOG_ERROR	LOG_ERR("\r\n[%s] : %u\r\n", __func__, __LINE__)
```

Comment:
When an error is encountered, the users should look at the driver source code to see what is wrong? There are no messages provided for them, only the function name and the line number.